commands/databases.go

```diff
@@ -481,6 +481,9 @@ To retrieve a list of your databases and their IDs, call `+"`"+`doctl databases
 	AddStringFlag(cmdDatabaseUserCreate, doctl.ArgDatabaseUserMySQLAuthPlugin, "", "",
 		"set auth mode for MySQL users")
 
+	CmdBuilderWithDocs(cmd, RunDatabaseUserResetAuth, "reset the mysql auth for <database-id> <user-name> <new-auth-mode>",
+		"Resets a users MySQL auth plugin", "This command resets the MySQL auth plugin for a given user. It will return the new user credentials. Valid auth plugin values are 'caching_sha2_password'and 'mysql_native_password'", Writer, aliasOpt("rs"))
```

Comment:
Missing a space here.
```suggestion
		"Resets a users MySQL auth plugin", "This command resets the MySQL auth plugin for a given user. It will return the new user credentials. Valid auth plugin values are 'caching_sha2_password' and 'mysql_native_password'", Writer, aliasOpt("rs"))
```